data/Templates/Ccda/Resource/_MedicationAdministration.liquid

```diff
@@ -13,8 +13,11 @@
         "status":"{{ medicationAdministration.statusCode.code }}",
         "effectivePeriod":
         {
-            "start":"{{ medicationAdministration.effectiveTime.low.value | format_as_date_time }}",
-            "end":"{{ medicationAdministration.effectiveTime.high.value | format_as_date_time }}",
+            {% for effectiveTime in medicationAdministration.effectiveTime -%}
```

Comment:
Is `medicationAdministration.effectiveTime` a list?

Reply:
Yes, it is a list.

Reply:
Thanks.

Reply:
Yes, I modify the code to use the first item of the list now.

Reply:
Got it, thanks.